synthtool/gcp/templates/node_library/.github/auto-label.yaml

```diff
@@ -0,0 +1,3 @@
+product: true
```

Comment:
did `product` default to true? wondering if we should just add requestsize, to make this change minimal.

Reply:
> did `product` default to true? wondering if we should just add requestsize, to make this change minimal.

Actually you right - let me remove it